app/components/ui/atoms/PersonPod.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import styled from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+import { Action, Button } from '@pubsweet/ui'
+import { Flex, Box } from 'grid-styled'
+
+import Icon from './Icon'
+
+const RegularP = styled.p`
```

Comment:
What is 'P' ?

Reply:
Ah - is it for "paragraph"? If so where its used maybe its better to name like `StatusStyle` or `KeywordStyle` ?

Reply:
Yeah I get what you mean @diversemix. But `RegularP` is being used in 3 places, so to use this naming, I would need to create 3 components: `NameStyle`, `InstitutionStyle` and `NonClickableKeywordStyle` all with identical styling on them. (`SmallP` is only used for keywords and could therefore become `KeywordStyle`)

Reply:
I do get the reasoning though - this naming is more explicit in the context of the `PersonPod`.

But if we're going to use these stylings (essentially just a `p` with a specified font size and line height) in other contexts than just the `PersonPod` component, then maybe having a generic name like `SmallParagraph` is better?

If you take a look at the top comment of this PR, I was thinking that maybe we should extract `RegularP` and `SmallP` out into their own atoms? After all, we have `Headings.js` for `h1`, `h2`, etc, which are styled according to our theme. Should we not have the same thing for `p` tags?

I should probably call it `RegularParagraph` instead of `RegularP` though, to avoid the "what is 'P'?" confusion? I agree that this is confusing.